server/backends/github/github.go

```diff
@@ -64,22 +66,18 @@ func AuthEnabled(env environment.Env) bool {
 	return *JwtKey != ""
 }
 
+type GithubStatusPayload = github.RepoStatus
```

Comment:
I changed this type to reference the type from `go-github` so that `interfaces.go` wouldn't need to depend on this package. It unfortunately caused a bit of noise throughout the PR because the fields are `*string` instead of `string`